[OSDOCS#17136]: Changes as per CQA guidelines (#111163)
openshift/openshift-docs: modules/microshift-viewing-audit-logs.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-security-context-constraints-alert-eval_{context}"]

> = Identify pod security violations through audit logs

Reviewer: revert

[role="_abstract"]

> Identify pod security admission violations in a workload by viewing the server audit logs. The following procedure describes how to access and parse audit logs to find these violations.

Reviewer: ```suggestion
You can identify pod security admission violations in a workload by viewing the server audit logs. To do this, you must access and parse audit logs to find these violations.
```
removing "the following" and the self-referential "procedure"

.Prerequisites

> * You have installed `jq` utility.

Reviewer: ```suggestion
* You have installed the `jq` utility.
```
add the article if you add the noun

* You have root access to the node.

.Procedure

. Retrieve the node name and assign it to a variable.
+
[source,terminal]
----
$ NODE_NAME=$(oc get node -ojsonpath='{.items[0].metadata.name}')
----

> . List the available audit logs.

Reviewer: revert; you removed the action and changed the meaning
